sycl/include/CL/sycl/detail/common.hpp

```diff
@@ -24,30 +24,46 @@
 __SYCL_INLINE_NAMESPACE(cl) {
 namespace sycl {
 namespace detail {
+
+#if !defined(NDEBUG) && (_MSC_VER > 1929 || __has_builtin(__builtin_FILE))
+#define __XPTI_FILE_NAME __builtin_FILE()
```

Comment:
I am not sure about the name of the macros because code_location can be used by different components.
Can we have __CLOC or __CODELOC prefix instead?

Reply:
Yes, we can. I will change it to __CODELOC.